src/components/Root.tsx

```diff
@@ -0,0 +1,27 @@
+import {
+  HashRouter as Router,
+  Routes,
+  Route,
+  Navigate,
+} from 'react-router-dom';
+import { App } from '../App';
+import { HomePage } from './HomePage/HomePage';
+import { PeoplePage } from './PeoplePage/PeoplePage';
+import { ErrorPage } from './ErrorPage/ErrorPage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="/home" element={<Navigate to="/" />} />
```

Comment:
You can move all routes in enum or constants and use them in all places in your application.